### PR TITLE
BoardConfig: use the exFAT driver from the kernel.

### PR DESCRIPTION
**Pull request by FacuM**
This pull request aims to fix the exFAT SD Card (or any other external device) support on this legacy device, disabling the broken fuseblk implementation.

### CODE REVIEW

File: BoardConfig.mk [hunk 1]
```diff
@@ -164,6 +164,8 @@ BOARD_ROOT_EXTRA_FOLDERS := firmware persist
 TARGET_USERIMAGES_USE_EXT4 := true
 TARGET_USERIMAGES_USE_F2FS := true
 TARGET_USES_MKE2FS := true
+TARGET_KERNEL_HAVE_EXFAT := true
+TARGET_EXFAT_DRIVER := exfat
```
Comment (adithya2306) [inline BoardConfig.mk]: Already present https://github.com/AOSP-LEGACY/device_lenovo_A6020/blob/staging/ten/BoardConfig.mk#L118

Comment (FacuM) [reply BoardConfig.mk]: > Already present https://github.com/AOSP-LEGACY/device_lenovo_A6020/blob/staging/ten/BoardConfig.mk#L118

`TARGET_KERNEL_HAVE_EXFAT := true` is needed, please do a conflict resolution before merging, or let me know if it'd be better if I send this PR without the `TARGET_EXFAT_DRIVER := exfat` flag.

Comment (adithya2306) [reply BoardConfig.mk]: > > Already present https://github.com/AOSP-LEGACY/device_lenovo_A6020/blob/staging/ten/BoardConfig.mk#L118
> 
> `TARGET_KERNEL_HAVE_EXFAT := true` is needed, please do a conflict resolution before merging, or let me know if it'd be better if I send this PR without the `TARGET_EXFAT_DRIVER := exfat` flag.

Resend

**Comment by FacuM** [comment]
@ghostrider-reborn Please re-check this PR now that I've updated the required branch.

**Comment by adithya2306** [comment]
I did a small research and it turns out that the flag `TARGET_KERNEL_HAVE_EXFAT` is not being used anywhere in the source. Can you show me some reference where it's being used?

**Comment by FacuM** [comment]
> I did a small research and it turns out that the flag `TARGET_KERNEL_HAVE_EXFAT` is not being used anywhere in the source. Can you show me some reference where it's being used?

Well, not on AOSP-Legacy, but a couple of ROMs use it, it's meant to be used as a way to make the tree common across all ROMs, as common as possible.

https://github.com/search?q=TARGET_KERNEL_HAVE_EXFAT&type=Commits

**Pull request merged**